storage/src/test/scala/coop/rchain/storage/LmdbTests.scala

```diff
@@ -355,33 +244,44 @@ object LmdbTests {
 
     try {
       for (i <- 0 until numKeys) {
-        print(s"put($i): ")
         val key = i.toString
         for (v <- 0 until valuesCount) {
           val value = (i + 1 + v).toString
-          print(value + ", ")
           lmdb.put(key, value)
         }
-        println()
       }
 
-      lmdb.displayRowsStrStr()
+      for (i <- 0 until numKeys) {
+        val valuesSeen = scala.collection.mutable.Map[String, Boolean]()
```

Comment:
Can probably optimize this code a lot using sets, i.e. create set of valuesSeen, then use set difference with values.